tests/scripts/thread-cert/Cert_5_8_03_KeyIncrementRollOver.py

```diff
@@ -86,38 +86,37 @@ def verify(self, pv):
 
         # Step 1: The DUT must start the network using
         # thrKeySequenceCounter = 127
-        _lpkts = leader_pkts.filter_mle_cmd(consts.MLE_ADVERTISEMENT).must_next()
+        _lpkts = leader_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
```

Comment:
```suggestion
        pkt = leader_pkts.filter_mle_cmd(MLE_ADVERTISEMENT).must_next()
```
`must_next` returns a single Packet.